packages/devtools/lib/src/memory/memory_controller.dart

```diff
@@ -113,12 +113,25 @@ class MemoryController {
 
     for (Map instance in instances) {
       final String objectRef = instance['id'];
-      result.add(InstanceSummary(classRef, objectRef));
+      result.add(InstanceSummary(classRef, className, objectRef));
     }
 
     return result;
   }
 
+  Future<Instance> getObject(String objectRef) async {
+    final Map params = {
+      'objectId': objectRef,
+    };
+    final Response response = await serviceManager.service.callMethod(
+      'getObject',
+      isolateId: _isolateId,
+      args: params,
+    );
+
+    return Instance.parse(response.json);
```

Comment:
We would need to handle errors and Sentinel objects. Hopefully just use the existing impl of this.

Reply:
Right, now that I'm call getObject.